README.md

```diff
@@ -59,7 +59,7 @@ for more information.
 # Quick Start
 
 See the [technical documentation](https://openexr.readthedocs.io) for
-complete details, but to get started, the "hello, world" `.exr` writer program is:
+complete details, but to get started, the "hello, world" `.exr` writer program `writer.cpp` is:
```

Comment:
Make this :download:`writer.cpp <src/writer/writer.cpp>`

Reply:
And add a similar link to :download:CMakeLists.txt <src/writer/CMakeLists.txt>

Reply:
Not sure how to do that MD change. The main thing here is the makefile below assumes this code is named writer.cpp. It would also be good to have steps on how to build it like how to install openexr, build the lib, then build this project.

Reply:
Right, `:download:` is a sphinx/rst format, but this is markdown. I see what you're saying, makes sense now.